src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +28,7 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.sequenceNumber = Person.nextSequenceNumber++;
```

Comment:
Since nextSequenceNumber is a class-level variable of Person, you can access it from within Person class without using the "Person." keyword.